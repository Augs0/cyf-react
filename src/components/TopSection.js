import React from 'react'

export const TopSection = ({title, content}) => {
  return (
    <div className="section-container top-section">
      <div className="container">
        <div className="row">
          <div className="col-sm-8 col-sm-offset-2 section-description">
            <h2>{title}</h2>
            <div className="divider-1">
              <span></span>
            </div>
            <div>{content ? content : 'Oops, did not get content'}</div>
          </div>
        </div>
      </div>
    </div>
  )
}