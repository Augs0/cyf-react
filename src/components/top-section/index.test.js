import React from 'react';
import { shallow } from 'enzyme';
import TopSection from './';

describe('TopSection', () => {
  it('renders correctly', () => {
    const content = <p>Testing 1 2 3!</p>;
    const title = 'Test title';
    expect(
      shallow(<TopSection title={title} content={content} />),
    ).toMatchSnapshot();
  });
});
