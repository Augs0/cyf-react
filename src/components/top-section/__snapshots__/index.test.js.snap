// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopSection renders correctly 1`] = `
<div
  className="section-container top-section"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-sm-8 col-sm-offset-2 section-description"
      >
        <h2>
          Test title
        </h2>
        <div
          className="divider-1"
        >
          <span />
        </div>
        <div>
          <p>
            Testing 1 2 3!
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
